sdv/multi_table/base.py

```diff
@@ -217,7 +217,8 @@ def auto_assign_transformers(self, data):
         """
         for table_name, table_data in data.items():
             self._validate_table_name(table_name)
-            self._table_synthesizers[table_name].auto_assign_transformers(table_data)
+            synthesizer = self._table_synthesizers[table_name]
+            self._skip_foreign_key_transformations(synthesizer, table_name, table_data)
```

Comment:
minor: could we rename `_skip_foreign_key_transformations` to make it clear that it also assigns the transformers for a table? Maybe just `_assign_table_transformers` or something along those lines

Reply:
Also @frances-h since you are calling it up here, it would be nice to move the method definition above this one